sdk/cosmos/azure-cosmos/dev_requirements.txt

```diff
@@ -1,2 +1,2 @@
 -e ../../../tools/azure-sdk-tools
--e ../../core/azure-core
+../../core/azure-core
```

Comment:
Can we remove this from the dev requirements completely now?

Reply:
That should not be part of a version PR, I guess it's fixing something else that was broken at that time.